dask/dataframe/io/parquet/arrow.py

```diff
@@ -181,7 +181,7 @@ def _frag_subset(old_frag, row_groups):
 
 
 def _collect_pyarrow_dataset_frags(
-    ds, filters, valid_paths, fs, split_row_groups, gather_statistics, read_from_paths
+    ds, filters, valid_paths, fs, split_row_groups, gather_statistics
```

Comment:
We are no longer making it possible to pass a fragment object in the graph. So, `read_from_paths` is always True now.

Reply:
`read_from_paths` is actually back in the code, but it is not needed here.